docs/source/modules/hello_world/hello_world.rst

```diff
@@ -234,6 +240,12 @@ Immediately after that, this will pop up:
 .. literalinclude :: celery.txt
    :language: text
 
+You may not see all of the info logs listed after the Celery C is displayed. If you'd like to see them you can change Merlin's log levels with the ``-lvl`` or ``--level`` tag. For more information on this, you can run:
```

Comment:
this is actually changed via the workers' log level, not merlin's

Reply:
Do you have an example of how to change their log levels via the CLI? When I run `merlin run-workers --help` the optional arguments don't show a log level flag to set. The only optional flags that are listed for run-workers are: --help, --worker-args, --steps, --echo, and --vars.

Reply:
@lucpeterson (not sure if it notifies you when I comment so figured I'd add this @ here)